cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -84,7 +84,7 @@ func (w *wizard) initializeOptions() []wizardOpt {
 	options := []wizardOpt{
 		{desc: "Get node status", invoke: func() { w.stats(w.orchestrator) }},
 		{desc: "View protocol parameters", invoke: w.protocolStats},
-		{desc: "List registered orchestrators", invoke: func() { w.registeredOrchestratorStats() }},
+		{desc: "List active orchestrators", invoke: func() { w.registeredOrchestratorStats() }},
```

Comment:
While you're entirely correct, I felt like active was more intuitive to the user. 

Knowing what pending means would require some deeper understanding of how the protocol keeps track of the working set. 

Have no preference here so I'll leave it up to you.

Reply:
The main reason I would avoid saying "active" here is to avoid the following confusing scenarios:
- An orchestrator will become active next round, sees itself in the displayed table and notices that its listed as inactive in the "Active" column
- An orchestrator will become inactive next round, does not see itself in the displayed table and believes that it is inactive when it is still active for the remainder of the round

At the same time, I agree that introducing the pending term might be confusing if there is not also an option to view the active orchestrators.

I think we should standardize the language used to describe the orchestrator set in all tools including the CLI and the explorer. How about we keep the term "registered" in this PR and update the language once we have finalized the language we want to use across all tools?

Reply:
I'm ok with keeping "registered" for now and formalizing a ubiquitous streamflow language across the board.

Reply:
dropped the commit